### DISCUSSION TITLE
TCC

### DISCUSSION THREAD

**Discussion by priscilavalverdes** [priscilavalverdes/TCC-Pos-abolicao#5] [Show and tell]
Alterei o cap. I.

@ericbrasiln , por favor quando você puder, você dá uma olhadinha. 
Veja se tem necessidade de colocar um trecho sobre a História Social.

**Comment by priscilavalverdes** [comment]
@ericbrasiln, sobre a parte específica do script de raspagem, você sugere que seja feito como no tcc? Mantendo a maneira descrita no relatório ou resumindo as etapas?

**Comment by priscilavalverdes** [comment]
Precisava de um sugestão também de como colocar as famílias de código... é para manter do jeito do relatório?

**Comment by priscilavalverdes** [comment]
@ericbrasiln, finalizei a metodologia.
Coloquei duas partes:
I- Tentei falar um pouco sobre fontes, não sei se as citações são pertinentes.
II - Coloquei o passo a passo em si.

**Reply by ericbrasiln** [reply]
Oi @priscilavalverdes 
Vou tentar ler com mais calma hoje a tarde, ok?

**Comment by priscilavalverdes** [comment]
Certo, @ericbrasiln . Até o final da próxima semana insiro o último capítulo com a análise dos resultados.

**Comment by priscilavalverdes** [comment]
@priscilavalverdes , no trecho que fala da consolidação do pós-abolição, é 2003 mesmo?

**Comment by priscilavalverdes** [comment]
Na verdade não me lembro de nenhuma referência tradicional que fale sobre isso, já aprendi a história desconstruída rsrs.

**Reply by priscilavalverdes** [reply]
Mas vou procurar.

**Comment by ericbrasiln** [comment]
@priscilavalverdes  Fiz os reviews e fiz os merges nos PR.
Posso ler novamente? Ou melhor esperar a inclusão do capítulo 3?

**Comment by priscilavalverdes** [comment]
Oi @ericbrasiln . Ainda falta incluir as alterações da parte II da metodologia. Se vc preferir, acho melhor esperar um pouco.

**Comment by ericbrasiln** [comment]
Bom dia, @priscilavalverdes 
Sabe o que percebi, você ainda não incluiu na monografia uma descrição de como está o processo de escrita: vc está usando o github para documentar todo o processo e preservar os dados. Isso é muito importante. Dá uma olhada naquela minha fala na FGV que eu acho que pode te ajudar a incluir essa questão.

Outra coisa: que tal colocarmos o projeto completo do atlas disponível tb, assim como uma pasta com todos os papers que vc leu? eu tenho uma versão do projeto do atlas exportado aqui, mas é a versão 8, e isso geraria uma incoerência com seu texto.
Acho que vc pode exportar a árvore de códigos em csv (ou xls) e o bundle completo do sei atlas. depois a gente coloca aqui no github (se o arquivo for muito grande, eu coloco na nuvem do LABHD e o link aqui).

O que acha?

**Reply by priscilavalverdes** [reply]
> Bom dia, @priscilavalverdes Sabe o que percebi, você ainda não incluiu na monografia uma descrição de como está o processo de escrita: vc está usando o github para documentar todo o processo e preservar os dados. Isso é muito importante. Dá uma olhada naquela minha fala na FGV que eu acho que pode te ajudar a incluir essa questão.
> 
Oi @ericbrasiln , bom dia.

Eu tinha em mente que coloquei alguma coisa sobre isso. Vou olhar e adiciono ou complemento.

> Outra coisa: que tal colocarmos o projeto completo do atlas disponível tb, assim como uma pasta com todos os papers que vc leu? eu tenho uma versão do projeto do atlas exportado aqui, mas é a versão 8, e isso geraria uma incoerência com seu texto. Acho que vc pode exportar a árvore de códigos em csv (ou xls) e o bundle completo do sei atlas. depois a gente coloca aqui no github (se o arquivo for muito grande, eu coloco na nuvem do LABHD e o link aqui).
> 
> O que acha?

Acho ótimo. Faremos.

Eu vou só revisar rapidinho o que escrevi ontem da reformulação da escrita das etapas de raspagem e construção do dataset e já commito.

**Comment by priscilavalverdes** [comment]
@ericbrasiln , fiz agora. à noite começo o cap III com as análises.

**Comment by priscilavalverdes** [comment]
Oi @ericbrasiln, bom dia!
Comecei a fazer os commits mas por favor, não leia ainda não pois vou commitar mais coisas ao longo do dia. Obrigada.

**Reply by ericbrasiln** [reply]
ok. só vou conseguir ler amanhã

**Comment by priscilavalverdes** [comment]
@ericbrasiln , ainda vou trocar a citação do zotero, melhorar o parágrafo sobre o github e adicionar no corpo do *.md* as listas e os gráficos.

**Comment by priscilavalverdes** [comment]
@ericbrasiln , vou deixar os agradecimentos aqui por enquanto.


Agradeço ao Tempo, que se encarrega de colocar tudo em seu devido lugar: pessoas, caminhos e sentimentos. Agradeço a minha mãe, Eliete Valverde, meu amor, que me ensinou a resistir e por quem me reinvento todos os dias. Agradeço a Eric Brasil, meu orientador, a quem posso chamar de amigo, por todo incentivo e por ter me ensinado o significado do pronome “nós” dentro da universidade. Sem ele, esta monografia não teria sentido. Agradeço aos professores Leonardo Nascimento e Paulo Fonseca do Laboratório de Humanidades Digitais da Ufba- LABHD, do qual me sinto feliz em fazer parte, pelo acolhimento e por terem feito parte da formação técnica. Sem o laboratório não teria sido possível desenvolver a pesquisa da maneira que se encontra. Agradeço a Ana Carolina Veloso, minha amiga, pela parceria e por junto com Eric ter me ensinado a trabalhar em equipe. Obrigada a todos vocês por terem construído junto comigo este trabalho!

**Reply by ericbrasiln** [reply]
Tá muito legal. 
Um conselho: agradeça tb à banca e à unilab. Faz parte, é de praxe.
Mas se  não quiser não coloque.

**Reply by priscilavalverdes** [reply]
> Tá muito legal. Um conselho: agradeça tb à banca e à unilab. Faz parte, é de praxe. Mas se não quiser não colocar.

Tá certo, vou colocar.

**Comment by priscilavalverdes** [comment]
OI @ericbrasiln  , fiz algumas correções, inclusive a que você havia marcado. Mas ainda estou com a leitura muito condicionada ao processo da escrita, provavelmente não estou vendo alguns erros. Daqui há dois dias volto a olhar novamente. Quando vc puder, por favor, você renderiza com o quarto.
